src/map.js

```diff
@@ -209,17 +209,29 @@ export default class GpxMap {
                 link.innerText = 'Download as SVG';
 
                 const scale = 2;
-                const left = this.map.getPixelOrigin().x * scale;
-                const top = this.map.getPixelOrigin().y * scale;
-                const width = this.map.getSize().x * scale;
-                const height = this.map.getSize().y * scale;
-                const bounds = leaflet.bounds([left, top], [left+width, top+height]);
+                const bounds = this.map.getPixelBounds();
+                bounds.min = bounds.min.multiplyBy(scale);
+                bounds.max = bounds.max.multiplyBy(scale);
+                const left = bounds.min.x;
+                const top = bounds.min.y;
+                const width = bounds.getSize().x;
+                const height = bounds.getSize().y;
 
                 let svg = leaflet.SVG.create('svg');
                 let root = leaflet.SVG.create('g');
 
                 svg.setAttribute('viewBox', `${left} ${top} ${width} ${height}`);
 
+                if (this.options.theme.indexOf('CartoDB.DarkMatter') === 0) {
```

Comment:
Let's skip this for now. My thought with the SVG export was that it was something you could import into inkscape or whichever and manipulate it further there so I'd prefer not to have a background.

Reply:
Reverted